digiwf-email-integration/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>digiwf-email-integration-parent</artifactId>
+        <groupId>io.muenchendigital.digiwf</groupId>
+        <version>0.0.1-SNAPSHOT</version>
```

Comment:
Change all project versions to 0.0.1 for first release

Reply:
Done